### DISCUSSION TITLE
react版本表单设计器添加控件报错

### DISCUSSION THREAD

**Discussion by wangchen-4118** [alibaba/formily#2100] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/57650817/131964599-f51608ed-1981-48cf-9945-937bf00cad7b.png)
![image](https://user-images.githubusercontent.com/57650817/131964678-63e79325-939b-4c83-9ef0-09aa615e5203.png)

**Comment by janryWang** [comment]
怎么复现的，录个视频看看

**Reply by wangchen-4118** [reply]
https://user-images.githubusercontent.com/57650817/131966050-f0d724ba-b663-4a06-8e78-12073b965c28.mp4

是接入的有问题吗？

**Reply by janryWang** [reply]
你是在本地跑的？

**Reply by wangchen-4118** [reply]
是的
